web/packages/teleport/src/UnifiedResources/Resources.tsx

```diff
@@ -26,37 +26,55 @@ import {
 } from 'teleport/components/Layout';
 import ErrorMessage from 'teleport/components/AgentErrorMessage';
 import useTeleport from 'teleport/useTeleport';
+import useStickyClusterId from 'teleport/useStickyClusterId';
+import AgentButtonAdd from 'teleport/components/AgentButtonAdd';
+import { useInfiniteScroll } from 'teleport/components/hooks/useInfiniteScroll';
+import { useUrlFiltering } from 'teleport/components/hooks';
 
-import { useResources } from './useResources';
```

Comment:
Can we also remove the useResources file altogether?

Reply:
🤦 whoops